tests/modules/individuals/utils.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+
+from tests.modules.sightings.resources import utils as sighting_utils
+
+increment = 0
+
+
+def simple_sighting_encounter(db, flask_app_client, user, individual_sex='female'):
```

Comment:
As discussed, this is what I think should now be replaced by the create_asset_group_and_sighting in PR361